src/ImageSharp/Processing/Extensions/Transforms/ImagingUtilities.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Buffers;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.Memory;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Processing
+{
+    /// <summary>
+    /// Defines extensions that allow the computation of image integrals on an <see cref="Image"/>
+    /// </summary>
+    public static class ImagingUtilities
+    {
+        /// <summary>
+        /// Apply an image integral. See https://en.wikipedia.org/wiki/Summed-area_table
```

Comment:
You can use `<see href="https://en.wikipedia.org/wiki/Summed-area_table"/>` to better work with intellisense docs.